aiida_pseudo/cli/install.py

```diff
@@ -231,6 +231,23 @@ def cmd_install_pseudo_dojo(version, functional, relativistic, protocol, pseudo_
                 msg = f'md5 of pseudo for element {element} does not match that of the metadata {md5}'
                 echo.echo_critical(msg)
 
+        adjusted_cutoff_elements = []
+        for stringency, str_cutoffs in cutoffs.items():
+            max_cutoff_wfc = max([str_cutoffs[element]['cutoff_wfc'] for element in str_cutoffs])
+            max_cutoff_rho = max([str_cutoffs[element]['cutoff_rho'] for element in str_cutoffs])
+            for element, cutoff in str_cutoffs.items():
+                if cutoff['cutoff_wfc'] <= 0:
+                    cutoffs[stringency][element]['cutoff_wfc'] = max_cutoff_wfc * 2.0
+                    adjusted_cutoff_elements.append(element)
+                if cutoff['cutoff_rho'] <= 0:
```

Comment:
Wonder if we should add a check that if we are adjusting the wfc cutoff, we also have to update the rho cutoff. At least that way we make sure that they stay consistent.

Reply:
Because I generate the `cutoff_rho` as `2.0 * cutoff_wfc` for the PAW potentials, I really only need to check if the `cutoff_wfc <= 0`